test/util.test.ts

```diff
@@ -190,3 +190,19 @@ test('test keyGet2Func', () => {
   expect(util.keyGet2Func('/baz', '/foo', 'bar')).toEqual('');
   expect(util.keyGet2Func('/foo/baz', '/foo', 'bar')).toEqual('');
 });
+
+test('test escapeAssertion', () => {
+  expect(util.escapeAssertion('r.attr.value == p.attr')).toEqual('r_attr.value == p_attr');
+  expect(util.escapeAssertion('r.attp.value || p.attr')).toEqual('r_attp.value || p_attr');
+  expect(util.escapeAssertion('r.attp.value && p.attr')).toEqual('r_attp.value && p_attr');
+  expect(util.escapeAssertion('r.attp.value >p.attr')).toEqual('r_attp.value >p_attr');
+  expect(util.escapeAssertion('r.attp.value <p.attr')).toEqual('r_attp.value <p_attr');
+  expect(util.escapeAssertion('r.attp.value +p.attr')).toEqual('r_attp.value +p_attr');
+  expect(util.escapeAssertion('r.attp.value -p.attr')).toEqual('r_attp.value -p_attr');
+  expect(util.escapeAssertion('r.attp.value *p.attr')).toEqual('r_attp.value *p_attr');
+  expect(util.escapeAssertion('r.attp.value /p.attr')).toEqual('r_attp.value /p_attr');
+  expect(util.escapeAssertion('!r.attp.value /p.attr')).toEqual('!r_attp.value /p_attr');
+  expect(util.escapeAssertion('g(r.sub, p.sub) == p.attr')).toEqual('g(r_sub, p_sub) == p_attr');
+  expect(util.escapeAssertion('g(r.sub,p.sub) == p.attr')).toEqual('g(r_sub,p_sub) == p_attr');
+  expect(util.escapeAssertion('(r.attp.value || p.attr)p.u')).toEqual('(r_attp.value || p_attr)p_u');
```

Comment:
could you add more cases, so like:
```
r_app1.value == p.value
r_app1.value == p_app1.value
rapp1.value == p.value
rapp1.value == papp1.value
```

Reply:
This matching method will be supported in the future！

Reply:
You are solving is this problem, why should you support this in the future?

Reply:
@nodece It looks like we can not read the policy type like `r_app1`? `sec + Model.getKeySuffix(i)` will return like `r1` `r2` and so on.
https://github.com/casbin/casbin.js/blob/72a658b43c7037b4e7ae4c956168c578e535d17c/src/model/model.ts#L64-L73

Reply:
@sagilio thanks for your point.

Reply:
Hi @Gabriel-403, thanks for your contribution, casbin cannot support the policy type like `r_app1`, so you need to reset `escapeAssertion` method, and append the `r1.value == p1.value` and `r2.value == p4.value` to test cases.